src/software/ai/hl/stp/tactic/pass_defender/pass_defender_fsm.cpp

```diff
@@ -78,5 +78,30 @@ void PassDefenderFSM::interceptBall(const Update& event)
             TbotsProto::MaxAllowedSpeedMode::PHYSICAL_LIMIT,
             TbotsProto::DribblerMode::MAX_FORCE, TbotsProto::BallCollisionType::ALLOW,
             AutoChipOrKick{AutoChipOrKickMode::OFF, 0}));
+        return;
     }
+    // something is wrong when this happens,
+    // in this special case, the robot should move away from the ball
+    // as to not violate the rule about covering more than 80% of the ball
+
+    // to avoid dividing by 0
+    Angle face_ball_orientation;
+    if ((ball.position() - robot_position).length() == 0)
+    {
+        face_ball_orientation = Angle().zero();
+    }
+    else
+    {
+        face_ball_orientation = (ball.position() - robot_position).orientation();
+    }
```

Comment:
I think we should make a ticket to move this logic into and handle it in the API, and track for codebase. I suspect this may be happening in a few more places. What do you think @nimazareian ?

Reply:
Are you referring to updating `Vector::orientation` such that it returns `Angle::zero` if its a zero-vector? That would be a good idea imo. It would be similar to `Vector::normalize`

https://github.com/UBC-Thunderbots/Software/blob/cf3b99e878db2aebf64cc083c3e74d9274dd5352/src/software/geom/vector.cpp#L52-L58

@TylerKerswell since this might be a bit off topic for your PR, could you create an issue and tag it with a TODO in your code that it should be updated?